.vscode/tasks.json

```diff
@@ -104,34 +87,25 @@
         "focus": false,
         "panel": "dedicated"
       },
-      "args": [
-        "run",
-        "lint"
-      ],
+      "args": ["run", "lint"],
       "isBackground": false
     },
     {
       "label": "Publish Extensions",
       "command": "./scripts/publish-workflow.sh",
       "type": "shell",
-      "args": [
-        "${input:circleCiId}"
-      ]
+      "args": ["${input:circleCiId}"]
     },
     {
       "label": "Watch",
       "command": "npm",
       "type": "shell",
       "presentation": {
-        "reveal": "silent",
```

Comment:
This is what's actually changed: watch task is revealed in the terminal tab. The other changes are code formatting changes from prettier